bigtable/writes/write_batch.go

```diff
@@ -0,0 +1,59 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+package writes
+
+// [START bigtable_writes_batch]
+import (
+	"context"
+	"encoding/binary"
+	"fmt"
+	"io"
+
+	"cloud.google.com/go/bigtable"
+)
+
+func writeBatch(w io.Writer, projectID, instanceID string, tableName string) error {
+	// projectID := "my-project-id"
+	// instanceID := "my-instance-id"
+	// tableName := "mobile-time-series"
+
+	ctx := context.Background()
+	client, err := bigtable.NewClient(ctx, projectID, instanceID)
+	if err != nil {
+		return fmt.Errorf("bigtable.NewAdminClient: %v", err)
+	}
+	tbl := client.Open(tableName)
+	columnFamilyName := "stats_summary"
+	timestamp := bigtable.Now()
+
+	muts := make([]*bigtable.Mutation, 2)
+	b := make([]byte, 8)
+	binary.BigEndian.PutUint64(b, uint64(1))
+	muts[0] = bigtable.NewMutation()
+	muts[0].Set(columnFamilyName, "connected_wifi", timestamp, b)
+	muts[0].Set(columnFamilyName, "os_build", timestamp, []byte("12155.0.0-rc1"))
+	muts[1] = bigtable.NewMutation()
+	muts[1].Set(columnFamilyName, "connected_wifi", timestamp, b)
+	muts[1].Set(columnFamilyName, "os_build", timestamp, []byte("12145.0.0-rc6"))
+
+	rowKeys := []string{"tablet#a0b81f74#20190501", "tablet#a0b81f74#20190502"}
```

Comment:
Where do these values come from? Same with in the other samples. Might be worth a comment.

Reply:
This is a sample dataset/schema that we will be using throughout our docs